$NetBSD$

Add JIDX contest. Submitted to upstream, but not included yet.

--- src/config.xml.orig	2013-04-20 21:51:36.000000000 +0900
+++ src/config.xml	2013-05-22 20:14:47.000000000 +0900
@@ -604,6 +604,8 @@
     <cabrillocontest field="8" type="HF">DARC-WAEDC-RTTY</cabrillocontest>
     <cabrillocontest field="8" type="HF">DARC-WAEDC-SSB</cabrillocontest>
     <cabrillocontest field="8" type="HF">IARU-HF</cabrillocontest>
+    <cabrillocontest field="8" type="HF">JIDX-CW</cabrillocontest>
+    <cabrillocontest field="8" type="HF">JIDX-SSB</cabrillocontest>
     <cabrillocontest field="9" type="HF">NA-SPRINT-CW</cabrillocontest>
     <cabrillocontest field="9" type="HF">NA-SPRINT-SSB</cabrillocontest>
     <cabrillocontest field="8" type="HF">NAQP-CW</cabrillocontest>
